tests/helpers/builders/order_builder.py

```
@@ -282,18 +296,39 @@ def sl(self) -> TriggerOrderBuilder:
         """Alias for stop_loss()."""
         return self.stop_loss()
 
+    def reduce_only(self, value: bool = True) -> TriggerOrderBuilder:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)